tools/mob_suite/mob_recon.xml

```diff
@@ -1,7 +1,10 @@
-<tool id="mob_recon" name="MOB-Recon" version="3.0.0">
+<tool id="mob_recon" name="MOB-Recon" version="@VERSION@+galaxy0">
   <description>Type contigs and extract plasmid sequences</description>
+  <macros>
```

Comment:
Awesome. Thanks for adding this.

One suggestion I can make is that since the `@VERSION@` is repeated in two different tools, you could move the definition to a separate file **macros.xml** (`<token name="@VERSION@">3.0.1</token>`). Then in each of these files you could add `<import>macros.xml</import>`. That way the version only needs to be defined once. You could even refactor further to only define the `<requirements>` section once too.

You can see the `mlst` tool for an example of how this could be done: https://github.com/galaxyproject/tools-iuc/tree/master/tools/mlst

Reply:
Great suggestion. Implemented. Now less typing in the future update releases. I love `macros` feature in Galaxy recipes